File: objc/Project5/Project5/ViewController.m
//
//  ViewController.m
//  Project5
//
//  Created by Jinwoo Kim on 2/11/21.
//

#import "ViewController.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    [super prepareForSegue:segue sender:sender];
    GameViewController *vc = (GameViewController *)segue.destinationViewController;
    if (vc == nil) return;
    
    vc.targetLanguage = [[self.language titleForSegmentAtIndex:self.language.selectedSegmentIndex] lowercaseString];
    vc.wordType = [[self.words titleForSegmentAtIndex:self.words.selectedSegmentIndex] lowercaseString];
}

@end

File: swift/Project5/Project5/ViewController.swift
//
//  ViewController.swift
//  Project5
//
//  Created by Jinwoo Kim on 2/10/21.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet var language: UISegmentedControl!
    @IBOutlet var words: UISegmentedControl!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        super.prepare(for: segue, sender: sender)
        guard let vc = segue.destination as? GameViewController else {
            return
        }
        vc.targetLanguage = language.titleForSegment(at: language.selectedSegmentIndex)!.lowercased()
        vc.wordType = words.titleForSegment(at: words.selectedSegmentIndex)!.lowercased()
    }
}

